FlashSpace/Features/Gestures/WorkspaceTransitionManager.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  WorkspaceTransitionManager.swift
+//
+//  Created for FlashSpace
+//  Copyright © 2025 Wojciech Kulik. All rights reserved.
+//
+
+import AppKit
+import SwiftUI
+
+final class WorkspaceTransitionManager {
```

Comment:
There is also one more problem with that approach. The app takes a screenshot on workspace change for Space Control, and when transitions are enabled, screenshots are a little darker because it's taken when the transition begins.